docs/detections/prebuilt-rules/rule-details/ntds-or-sam-database-file-copied.asciidoc

```diff
@@ -72,6 +75,19 @@ process.pe.original_file_name in ("Cmd.Exe", "PowerShell.EXE",
 [[ntds-or-sam-database-file-copied-history]]
 ==== Rule version history
```

Comment:
I've pushed some changes to mappings in https://github.com/elastic/detection-rules/pull/1495, I don't know if these changes should be included here

Reply:
they will reflect in the next release - the docs are generated from what is staged at kibana.

Also, any changes we need to make here to the rules themselves we will do in the rules repo

Reply:
Got it